app/src/context/Exchange.tsx

```diff
@@ -16,6 +16,14 @@ export type ExchangeValue = {
   ) => Promise<void>;
   gpibUserID: string | undefined;
   reset: () => void;
+  verifyOnExchange: (
+    userName: string | undefined,
+    password: string | undefined,
+    firstName: string | undefined,
+    lastName: string | undefined,
+    yob: string | undefined
```

Comment:
can we define a type for this, so each other 3rd party can use it.
also why can be undefined?

Reply:
because values come from useClaimValue function, which can return undefined